xds/src/main/java/com/linecorp/centraldogma/xds/k8s/v1/XdsKubernetesEndpointFetchingService.java

```diff
@@ -241,24 +236,41 @@ private static class KubernetesEndpointsUpdater {
                     if (endpoints.isEmpty()) {
                         return;
                     }
-                    executorService.execute(this::maybeSchedule);
+                    executorService.execute(this::maybePushK8sEndpoints);
                 }, true);
             }));
         }
 
-        void maybeSchedule() {
-            if (closing || scheduledFuture != null) {
+        private void maybePushK8sEndpoints() {
+            if (closing) {
                 return;
             }
-            // Commit after 3 seconds so that it pushes with all the fetched endpoints in the duration
-            // instead of pushing one by one.
```

Comment:
I understand that the debound logic has been removed.

Reply:
That's correct because the way that `KubernetesEndpointGroup` fetches endpoints is changed. 😉